osu.Game/Screens/Edit/Setup/SetupSection.cs

```diff
@@ -59,7 +59,7 @@ protected SetupSection()
                     {
                         RelativeSizeAxes = Axes.X,
                         AutoSizeAxes = Axes.Y,
-                        Spacing = new Vector2(20),
+                        Spacing = new Vector2(10),
```

Comment:
Decreased this because items were starting to look too spaced-out with the metadata.